.github/workflows/pull_request.yml

```diff
@@ -33,4 +33,4 @@ jobs:
         distribution: 'adopt'
         cache: maven
     - name: Build with Maven
-      run: mvn -B verify --file pom.xml
+      run: mvn -B verify --file pom.xml -Dsurefire.useFile=false -DdisableXmlReport=true
```

Comment:
The flag ``disableXmlReport`` is deprecated, see [the documentation](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#disableXmlReport).